# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fbterm/hotfix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fbterm-1.0/src/screen.cpp.vanilla	2008-07-06 12:02:14.000000000 +0200
+++ fbterm-1.0/src/screen.cpp	2008-07-06 12:07:00.000000000 +0200
@@ -80,7 +80,7 @@
 	s32 devFd;
 	for (u32 i = 0; i < FB_MAX; i++) {
 		snprintf(devname, 32, "/dev/fb%d", i);
-		devFd = open(devname, O_RDWR | O_CLOEXEC);
+		devFd = open(devname, O_RDWR);
         if (devFd >= 0) break;
 	}
 
